src/dynamics/video_player/player/player.js

```diff
@@ -811,7 +812,7 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
                         if (width) styles.width = isNaN(width) ? width : parseFloat(width).toFixed(2) + "px";
                         containerStyles = floatingonly ? {} : Objs.extend({}, styles);
                         if (!gallerySidebar && showsidebargallery && layout === "desktop" && !fullscreened)
-                            containerStyles.aspectRatio = this.get("sidebaroptions.aspectratio") || 838 / 360;
+                            if (!outstream) containerStyles.aspectRatio = this.get("sidebaroptions.aspectratio") || 838 / 360;
```

Comment:
Hi @Josewd, do we really need to add this condition? The value of `gallerySidebar` is already based on `outstream` based on changes to line 664.